packages/flutter_tools/test/general.shard/base/bot_detector_test.dart

```diff
@@ -44,14 +44,6 @@ void main() {
     });
 
     group('isRunningOnBot', () {
-      testWithoutContext('returns false unconditionally if BOT=false is set', () async {
```

Comment:
Instead of deleting this test could you update it to confirm that setting BOT to any string makes `isRunningOnBot` true? and having it be null leads to `false`